sdk/core/Microsoft.Extensions.Azure/src/Microsoft.Extensions.Azure.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;$(RequiredTargetFrameworks)</TargetFrameworks>
```

Comment:
Is this needed now that you've marked the project as `IsClientLibrary`?    It looks like `Directory.Build.Data.props (line 53)` adds `netstandard2.0` as a common target.

Reply:
My bad, debugging leftovers.